### PR TITLE
contacts + add task

### PR DESCRIPTION
**Pull request by adriandumavitagroup**

### CODE REVIEW

**Review by sandorbanvitagroup — DISMISSED** [review]

File: src/app/_models/common.ts [hunk 1]
```diff
@@ -81,6 +81,7 @@ export interface NavItem {
 export type InfoBarType =
   | 'CASE'
   | 'EVENT'
+  | 'EVENT_P'
```
Comment (danielrus) [inline src/app/_models/common.ts]: EVENT_P stands for event participant?
it's not clear from the name

File: src/app/events/event-components/event-participants-profile/event-participants-profile.component.ts [hunk 2]
```diff
@@ -51,4 +51,8 @@ export class EventParticipantsProfileComponent implements OnInit {
       complete: () => {},
     });
   }
+
+  public getParticipant(): EventParticipantDto {
```
Comment (danielrus) [inline src/app/events/event-components/event-participants-profile/event-participants-profile.component.ts]: do we use this anywhere?

File: src/app/shared/dynamic-form/dynamic-form.component.ts [hunk 3]
```diff
@@ -109,7 +112,11 @@ export class DynamicFormComponent implements OnInit, OnDestroy {
   }
 
   updateFormRawValueWithObjects(): any {
-    const rawValueTmp: any = {};
+    const rawValueTmp: any = {
+      assigneeUser: {
+        uuid: 'S3ROT2-XAXJYF-VMIN7W-NA5ASJ7U',
```
Comment (danielrus) [inline src/app/shared/dynamic-form/dynamic-form.component.ts]: is this necessary? the call should work without it, no?

File: src/variables.scss [hunk 4]
```diff
@@ -17,6 +17,7 @@ $grid-breakpoints: (
 );
 
 $primary: mat-color($sormas-primary, 700);
+$primary-light: #6bbfff;
```
Comment (danielrus) [inline src/variables.scss]: this should use the palette colors -> primary 400

File: src/app/_constants/enums.ts [hunk 5]
```diff
@@ -384,10 +384,29 @@ export enum ConveyanceTypeOptions {
...
-  CONTACT = 'Contact',
-  EVENT = 'Event',
-  GENERAL = 'General',
+  CASE = 'CASE',
+  CONTACT = 'CONTACT',
+  EVENT = 'EVENT',
+  GENERAL = 'GENERAL',
+}
+
+export enum TracingApp {
```
Comment (danielrus) [inline src/app/_constants/enums.ts:393]: is this used anywhere?

File: src/app/_constants/enums.ts [hunk 6]
```diff
@@ -384,10 +384,29 @@ export enum ConveyanceTypeOptions {
...
+  CORONAWARNAPP = 'Corona warn app',
+  OTHER = 'Other',
+  UNKNOWN = 'Unknown',
+}
+
+export enum TypeOfContact {
```
Comment (danielrus) [inline src/app/_constants/enums.ts:399]: is this used?

File: src/app/_models/common.ts (hunk 1 above)
Comment (adriandumavitagroup) [reply src/app/_models/common.ts]: fixed and pushed

File: src/app/events/event-components/event-participants-profile/event-participants-profile.component.ts (hunk 2 above)
Comment (adriandumavitagroup) [reply src/app/events/event-components/event-participants-profile/event-participants-profile.component.ts]: removed

File: src/app/shared/dynamic-form/dynamic-form.component.ts (hunk 3 above)
Comment (adriandumavitagroup) [reply src/app/shared/dynamic-form/dynamic-form.component.ts]: removed. It was hardcoded only for the demo. My bad

File: src/variables.scss (hunk 4 above)
Comment (adriandumavitagroup) [reply src/variables.scss]: fixed and pushed

File: src/app/_constants/enums.ts (hunk 5 above)
Comment (adriandumavitagroup) [reply src/app/_constants/enums.ts:393]: Yes. In contact-details-form-data.ts

File: src/app/_constants/enums.ts (hunk 6 above)
Comment (adriandumavitagroup) [reply src/app/_constants/enums.ts:399]: Yes. In contact-details-form-data.ts

**Review by danielrus — APPROVED** [review]

**Pull request merged**